docs/policy-only.md

```diff
@@ -24,7 +24,7 @@ incoming traffic is received on this PtP device. This is a spoke-and-hub model,
 traffic, even within the same worker Node must traverse first to the host network and be
```

Comment:
Would you mention in policyOnly mode works only with Antrea Proxy but does not support kube-proxy? Do we have somewhere to mention the same for noEncap mode?

Reply:
Antrea-proxy to my knowledge has not been mentioned in any docs in the repo, do u want to mention it here, and say something like,  (... to be added later ...) ?

Reply:
It is mentioned in windows docs, but yes no separate doc. But worth to clarify policyOnly mode does not work with kube-proxy?

Reply:
mentioned Antrea-proxy in line 13.